src/molecules/select/Select.html

```diff
@@ -1,19 +1,22 @@
-<div class="select">
+<div class="a-select">
   <!-- @slot Label content (Named slot) -->
   <slot name="label">
     <a-label
       :for="id"
-      class="select__label"
+      class="a-select__label"
       v-if="label"
     >
       {{ label }}
     </a-label>
   </slot>
-  <div class="select__field">
+  <div :class="[
+      'a-select__field',
+      icon && 'a-select__field--icon'
+    ]">
```

Comment:
Indentation

Reply:
I moved it in one line, bc it's short :blush: